tests/models/reading_comprehension/bidirectional_attention_test.py

```diff
@@ -49,3 +50,13 @@ def test_trains_and_loads_correctly(self):
         # TODO(matt): fix the randomness that occurs here.
         # assert_allclose(model.model.predict(validation_input),
         #                 loaded_model.model.predict(validation_input))
+
+    def test_get_best_span(self):
+        # Note that the best span cannot be (1, 1) since even though
+        # 0.3 * 0.5 is the greatest value, the end span index is constrained
+        # to occur after the begin span index.
+        span_begin_probs = numpy.array([0.1, 0.3, 0.05, 0.3, 0.25])
+        span_end_probs = numpy.array([0.5, 0.1, 0.2, 0.05, 0.15])
+        begin_end_idxs = BidirectionalAttentionFlow.get_best_span(span_begin_probs,
+                                                                  span_end_probs)
+        assert begin_end_idxs == (1, 3)
```

Comment:
nope. The max is `0.3 (index 1 of start)` * `0.2 (index 2 of start)`. while this seems like it would be `(1, 2)`, the end index is purposefully incremented by one.

oops i just read your other comment, yes it would be `(1,2)` if we were doing inclusive span end. Does that answer your question?